source/StigData/Processed/WindowsServer-2019-DC-2.3.xml

```diff
@@ -3706,6 +3708,7 @@ Inherited from - "None" for all
 Principal - Access - Applies to
 SYSTEM - Full Control - This key and subkeys
 Administrators - Special - This key and subkeys
+Server Operators â€“ Read â€“ This Key and subkeys    (Domain controllers only)
```

Comment:
We need to make sure Server 2019 Domain Controller has this new User Right Assignment - Server Operators - Read - This Key and Subkeys..